src/components/NodesTable/NodesTableItem.vue

```diff
@@ -143,6 +142,18 @@ export default {
   }
   &__status-icon {
     margin-inline-start: 4px;
+    &--green {
+      color: rgb(61, 209, 151) !important;
+    }
+    &--red {
+      color: rgb(237, 82, 112) !important;
+    }
+    &--grey {
+      color: #9e9e9e !important;
+    }
+    &--orange {
+      color: rgb(248, 160, 97) !important;
+    }
```

Comment:
Remove this. You must specify colors only in `.v-theme--light` and `.v-theme--dark` selectors.